python/dataproc_templates/bigquery/README.md

```diff
@@ -0,0 +1,20 @@
+## BigQuery To GCS 
+
+General Execution:
+
+```
+export GCP_PROJECT=<project_id>
+export SUBNET=<region>
```

Comment:
subnet ?
lets try without subnet, if it works.

Reply:
responded on previous comment